index.html

```diff
@@ -155,7 +155,7 @@
         </td>
         <!-- Location -->
         <td>
-          <i class="location-button map icon"></i> <a href="https://garlandtools.org/db/#{{?it.data.location.spearfishing}}node{{??}}fishing{{?}}/{{=it.data.location.id}}"
+          <i class="location-button map icon"></i> <a href="https://ffxivteamcraft.com/db/en/{{?it.data.location.spearfishing}}spearfishing{{??}}fishing{{?}}-spot/{{=it.data.location.id}}"
```

Comment:
I would prefer not to hardcode the language...
There's currently a function in [viewmodel.js](https://github.com/icykoneko/ff14-fish-tracker-app/blob/2bd0baae5fcbb8894e5de42656ef53c02d3e6635/js/app/viewmodel.js#L274-L301) that returns links to the different sites, but it's for the fish or bait.

Not to make it worse, but..., since the template is only run when building a new entry in the table, if the user changes the language without refreshing the page, the language will remain pointing at the original one. Fortunately, there's another function in [layout.js](https://github.com/icykoneko/ff14-fish-tracker-app/blob/2bd0baae5fcbb8894e5de42656ef53c02d3e6635/js/app/layout.js#L56) that handles these.

So, there'd really need to be another function in FishEntry that returns link to location, `getLocationExternalLink` or something like that.  You'd have access to the information needed to tell if it's spearfishing or not this way too.  Just remember that intuition entries don't include the location.

Reply:
Teamcraft database links resolve to either the browser's language or the language you have set on the site, so in this case the `en` appears to be purely cosmetic. Do you think changes are still required if there's no functional benefit?

Reply:
Oh, if that's the case nah.  I was just worried someone's viewing my site in Japanese, and suddenly TC switches over to English on them haha.

Reply:
All good. I considered mentioning it in my PR because the hard coded locale rubbed me the wrong way as well. Guess I should have. :)